### DISCUSSION TITLE
New to everything!

### DISCUSSION THREAD

**Discussion by nacho365** [community/community#22583] [New to GitHub]
<p>Hi All, </p>
<p>I am new to programming and new to Github.</p>
<p>I have little knowledge of HTML etc. </p>
<p>I have purchased lots of tutorials on UDEMY and <a href="http://LYNDA.com" rel="nofollow noopener">LYNDA.com</a> to help me along the way. </p>
<p>I have also found this forum for my bookmarks. </p>
<p>I want to create a warehouse inventory system for my logistics company. </p>
<p>I was wondering how github worked, is there a way I can clone someone elses project and learn from that / amend and add over time. </p>
<p>I have downloaded ATOM to my desktop and also have sights on downloading PYTHON &amp; DJANGO. </p>
<p>I would appreciate some guidance of what I need to do.</p>
<p>thank you </p>
<p>Jason</p>

**Comment by Wabri** [comment, marked as answer]
<p>Ok, i try to answer to you with this list:</p>
<ol>
<li>how to learn github:</li>
</ol>
<ul>
<li><a href="https://github.community/t/how-to-quickly-learn-to-use-git-and-github/1521">github-beginners</a></li>
</ul>
<ol start="2">
<li>how to understand github workflow:</li>
</ol>
<ul>
<li><a href="https://guides.github.com/introduction/flow/" rel="nofollow noopener">introduction</a></li>
<li><a href="https://nvie.com/posts/a-successful-git-branching-model/" rel="nofollow noopener">git-branching-model</a></li>
</ul>
<ol start="3">
<li>how to improve your skills with python, git and github:</li>
</ol>
<ul>
<li>simply search on github a repository you may like, fork, develop something new on it and than pull request</li>
<li>create a new repository of your own and share</li>
</ul>
<ol start="4">
<li>for python and django you can find a lot of courses, online documentations and even on books</li>
<li>The last but not the least: never stop studing</li>
</ol>
<p>If you want to ask something else, do it :slight_smile:</p>

**Comment by nacho365** [comment]
<p>thank you Wabri, this is my bookmark!</p>
<p>thank you so much! </p>
